### PR TITLE
50 Create comments model

### PR DESCRIPTION
**Pull request by LinhNG274**
http://192.168.0.18:3333/issues/50

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :commentable, polymorphic: true
+  validates :content, presence: true, length: { minimum: 300 }
```
Comment (ngocnguyenbk) [inline app/models/comment.rb]: minimum 300 ?

**Review by ngocnguyenbk — COMMENTED** [review]
Oke

**Review by ngocnguyenbk — APPROVED** [review]
Oke

File: app/models/user.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   has_many :work_experiences, dependent: :destroy
+  has_many :comments, dependent: :destroy
```
Comment (N-Viet-ruby-dev) [inline app/models/user.rb]: Tham khảo https://guides.rubyonrails.org/association_basics.html#polymorphic-associations

**Review by luongvietdung — APPROVED** [review]
LGTM

**Pull request merged**